package.json

```diff
@@ -3,6 +3,8 @@
     "@datadog/browser-logs": "^4.50.0",
     "@datadog/browser-rum": "^4.50.0",
     "@emotion/react": "11.11.1",
+    "@lexical/history": "^0.11.3",
+    "@lexical/react": "^0.11.1",
```

Comment:
Don't we want to use exact versions here as well?

Reply:
good call 👌